frontend/src/generate-static-html.js

```diff
@@ -1,24 +1,42 @@
 
```

Comment:
Figuring out how this module gets used twisted my melon a bit:

* [We generate `frontend/build/static/app/generate-static-html.js` via browserify](https://github.com/mozilla/testpilot/pull/2351/files#diff-2001630d3abeb76e05328a0b74f00a76R61)
* This bundle gets deployed to the servers - but we don't actually use it anywhere on the site
* Instead, this is [a library for *gulp tasks*](https://github.com/mozilla/testpilot/pull/2351/files#diff-1f5c2f0f3cd2018239cc9819551e5693R65) to do the static React generation things

I'd say this stuff more properly belongs in `frontend/tasks/util` except that it needs babel & browserify preprocessing to summon React powers into the gulp context. It probably shouldn't be shipped to the web along with the other app JS, though.